conans/model/conf.py

```diff
@@ -61,6 +61,7 @@
     "tools.build:sysroot": "Pass the --sysroot=<tools.build:sysroot> flag if available. (None by default)",
     "tools.build.cross_building:can_run": "(boolean) Indicates whether is possible to run a non-native app on the same architecture. It's used by 'can_run' tool",
     "tools.build.cross_building:cross_build": "(boolean) Decides whether cross-building or not regardless of arch/OS settings. Used by 'cross_building' tool",
+    "tools.build.cross_building:native": "(boolean) Decides, when it is a cross-building, whether to use an isolated build context in the toolchains (MesonToolchain for now).",
```

Comment:
Do we really need this? Isn't this controlled by the ``native:True`` inside meson:build?

Reply:
Yes, that's the question. It was only to prevent users from unexpected errors, but it could also work as expected, even passing both files.